src/CommunityToolkit.Maui.Core/Views/Popup/PopupView.macios.cs

```diff
@@ -139,4 +173,87 @@ public override void Draw(CGRect rect)
 			return path.CGPath;
 		}
 	}
+}
+
+static class UIViewExtensions
```

Comment:
I think it's definitely should be in a separate file

Reply:
I kept getting build errors when putting it in the `Extensions` folder. 

I'll give it another shot 👍

Reply:
@brminnick Let me know if you still have issues with this one. I can take a look on my end

Reply:
Done ✅ 

(I forgot to add the using declarations in the new file last tie 🤦‍♂️)